### PR TITLE
docs: add more documentation to the helm charts 

### LINKED ISSUE #353: Add more documentation to the helm charts
Right now, the EDC team provides different EDC types
- [tractusx-connector-azure-vault](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/charts/tractusx-connector-azure-vault)
- [tractusx-connector-memory](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/charts/tractusx-connector-memory)
- [tractusx-connector](https://github.com/eclipse-tractusx/tractusx-edc/tree/main/charts/tractusx-connector)

Within each of these connectors, a helmchart and installation manual is provided. Because of the used daps it is not possible to run the `helm install` command, cause there are additional parameters needed.

These parameters should be added to the installation guide.

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## WHAT

Adds a bit of documentation to all the helm charts.

## WHY

QGate requirement. also, documentation is good!

## FURTHER NOTES

_List other areas of code that have changed but are not necessarily linked to the main feature. This could be method signature changes, package declarations, bugs that were encountered and were fixed inline, etc._

Closes #353

### CODE REVIEW

**Review by stephanbcbauer — APPROVED** [review]
LGTM

File: charts/tractusx-connector-azure-vault/README.md.gotmpl [hunk 1]
```diff
@@ -8,13 +8,46 @@
...
+
+The following requirements must be met before launching the application:
+
+- Write access to an Azure KeyVault instance is required to run this chart
+- Secrets are seeded in advance
+- the vault's client id, client secret, tenant id and vault name (not the url!) are known
+
+Please also consider using [this example configuration](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-tests/deployment/src/main/resources/helm/tractusx-connector-azure-vault-test.yaml)
+to launch the application.
+Combined, this run this shell command to start the in-memory Tractus-X EDC runtime:
```
Comment (florianrusch-zf) [inline charts/tractusx-connector-azure-vault/README.md.gotmpl:36]:
```suggestion
Combined, run this shell command to start the in-memory Tractus-X EDC runtime:
```

File: charts/tractusx-connector-azure-vault/README.md.gotmpl [hunk 2]
```diff
@@ -8,13 +8,46 @@
...
+export DAPS_CERT="$(cat daps.cert)"
+```
+
+## Launching the application
+
+The following requirements must be met before launching the application:
+
+- Write access to an Azure KeyVault instance is required to run this chart
+- Secrets are seeded in advance
+- the vault's client id, client secret, tenant id and vault name (not the url!) are known
```
Comment (florianrusch-zf) [inline charts/tractusx-connector-azure-vault/README.md.gotmpl:32]:
```suggestion
- The vault's client id, client secret, tenant id and vault name (not the url!) are known
```

File: charts/tractusx-connector-memory/README.md.gotmpl [hunk 3]
```diff
@@ -8,13 +8,39 @@
...
+
+## Launching the application
+
+The in-memory vault can be seeded directly with secrets that are passed in `<key>:<value>;<key2>:<value2>;...` format.
+This config value can be passed to the runtime using the `vault.secrets` parameter. In addition, the runtime requires a
+couple of configuration parameters, all of which can be found in the section below. Please also consider using
+[this example configuration](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/charts/tractusx-connector-memory/example.yaml)
+to launch the application.
+
+Combined, this run this shell command to start the in-memory Tractus-X EDC runtime:
```
Comment (florianrusch-zf) [inline charts/tractusx-connector-memory/README.md.gotmpl:33]:
```suggestion
Combined, run this shell command to start the in-memory Tractus-X EDC runtime:
```

File: charts/tractusx-connector/README.md.gotmpl [hunk 4]
```diff
@@ -8,11 +8,36 @@
...
+## Launching the application
+
+The following requirements must be met before launching the application:
+
+- Write access to a HashiCorp Vault instance is required to run this chart
+- Secrets are seeded in advance
+
+Please also consider using [this example configuration](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-tests/deployment/src/main/resources/helm/tractusx-connector-test.yaml)
+to launch the application.
+Combined, this run this shell command to start the in-memory Tractus-X EDC runtime:
```
Comment (florianrusch-zf) [inline charts/tractusx-connector/README.md.gotmpl:35]:
```suggestion
Combined, run this shell command to start the in-memory Tractus-X EDC runtime:
```

**Review by florianrusch-zf — CHANGES_REQUESTED** [review]

**Comment by paullatzelsperger** [comment]
> please regenerate the README.md's 😉

forgot to push - sry

**Review by florianrusch-zf — APPROVED** [review]

**Pull request merged**